web/src/js/index.js

```diff
@@ -1,473 +1,479 @@
+    }
+  },
+});
+
+function sendApiRequest(formURL, requestMethod) {
+  fetch(API_CRUD_ALLOWLIST + "?" + formURL, {
+    method: requestMethod,
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      refreshAllowlist();
+      app.closeAllowlistInsertPopup();
+      app.closeAllowlistDeletePopup();
+
+      iziToast.success({
+        color: "#3FBF61",
+        message: data.message,
+        messageColor: "white",
+      });
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+      });
+    });
+}
+
+// Get execution log for a single instance
+function getExecutionLog(executionLogUrl) {
+  app.showExecutionLogPopup = true;
+  app.showExecutionLogLoadingGif = true;
+
+  fetch(API_EXECLOG + executionLogUrl, {
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      app.executionLogTable = data["response"]["body"];
+      app.executionLogKey = decodeURIComponent(executionLogUrl);
+
+      setTimeout(function () {
+        app.executionLogDataTables = $("#execution-log-table").DataTable({
+          autoWidth: true,
+          columnDefs: [
+            { orderable: false, targets: [6] },
+            { className: "dt-center", targets: [6] },
+          ],
+          dom: "lrti",
+          paging: false,
+        });
+        app.showExecutionLogLoadingGif = false;
+      }, 10);
+
+      // Get execution mode
+
+      if (data["response"]["body"][0][7] == "True") {
+        app.executionLogMode = "Dry Run";
+      } else {
+        app.executionLogMode = "Destroy";
+      }
+
+      for (const log of data["response"]["body"]) {
+        // action taken
+        app.executionLogActionStats[log["5"]] =
+          ++app.executionLogActionStats[log["5"]] || 1;
+
+        // service and resource
+        app.executionLogServiceStats[log["2"] + " " + log["3"]] =
+          ++app.executionLogServiceStats[log["2"] + " " + log["3"]] || 1;
+
+        // region
+        app.executionLogRegionStats[log["1"]] =
+          ++app.executionLogRegionStats[log["1"]] || 1;
+      }
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+      });
+    });
+}
+
+// Get execution logs list
+function getExecutionLogList() {
+  app.showExecutionLogListLoadingGif = true;
+  fetch(API_EXECLOG, {
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      app.executionLogList = data["response"]["logs"].map((row) => {
+        let logDate = new Date(row["date"] + " UTC");
+        let localDate = logDate.toString().split(/ GMT/)[0];
+
+        row["key_escape"] = encodeURIComponent(row["key"]);
```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792602)
```suggestion
        row["key_escape"] = encodeURIComponent(row["key"]);
```